packages/backend/src/managers/playgroundV2Manager.ts

```diff
@@ -206,7 +231,11 @@ export class PlaygroundV2Manager implements Disposable {
         });
       })
       .catch((err: unknown) => {
-        console.error('Something went wrong while creating model reponse', err);
+        console.error('Something went wrong while creating model response', err);
+        this.telemetry.logError('playground.error', {
```

Comment:
Thanks,

FYI could you avoid the usage of telemetry.logError (we've stopped to use that pattern)


prefer

```ts
telemetryData = {};
try {
  const foo = await foo();
  telemetryData.foo =  foo
} catch (error) {
  telemetryData.error = error
}

telemetryLogger.logUsage('myAction', telemetryData);
```

it's because it's easier to map errors with a single event rather than having to match stuff from 2 different events

so if you can send the error as part of the 'playground.submit' event (if any)

Reply:
Oh thanks for the information, I will fix it, and be careful !